src/sort/add_primitive/src/lib.rs

```diff
@@ -182,6 +191,7 @@ pub fn add_primitive(input: TokenStream) -> TokenStream {
     // This is the big `quote!` block that ties everything together.
     quote! {{
         #[allow(unused_imports)] use ::egglog::{*, constraint::*};
+        #[allow(unused_imports)] use ::std::any::TypeId;
```

Comment:
I also noticed that Span and ExecutionState are used but not imported, so users of this macro may potentially get a type error.